ddl/ddl.go

```diff
@@ -109,6 +109,7 @@ var (
 	errTooManyFields            = terror.ClassDDL.New(codeTooManyFields, "Too many columns")
 	errInvalidSplitRegionRanges = terror.ClassDDL.New(codeInvalidRanges, "Failed to split region ranges")
 	errReorgPanic               = terror.ClassDDL.New(codeReorgWorkerPanic, "reorg worker panic.")
+	ErrGeneratedColumnFunctionIsNotAllowed               = terror.ClassDDL.New(codeErrGeneratedColumnFunctionIsNotAllowed, "Expression of generated column '%s' contains a disallowed function.")
```

Comment:
we should add this error message to the parser repo?

Reply:
yes, done in https://github.com/pingcap/parser/pull/194